File: C++/ArrayAddition.cpp
#include <iostream>
using namespace std;

int main() {
    int maxSize;

    cout << "enter array size: " << endl;
    cin >> maxSize;

    int arrA[maxSize], arrB[maxSize], arrC[maxSize];

    cout << "enter array A elements: " << endl;
    for (int i = 0; i < maxSize; i++) {
        cin >> arrA[i];
    }

    cout << "enter array B elements: " << endl;
    for (int j = 0; j < maxSize; j++) {
        cin >> arrB[j];
    }

    for (int m = 0; m < maxSize; m++) {
        arrC[m] = arrA[m] + arrB[m];
    }

    cout << "Resultant Array: " << endl;
    for (int x = 0; x < maxSize; x++) {
        cout << arrC[x] << " " << endl;
    }

    return 0;
}

File: Java/ArrayAddition.java
import java.util.Scanner;

class ArrayAddition {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("enter array size: ");
        int maxSize = sc.nextInt();

        int[] arrA = new int[maxSize];
        int[] arrB = new int[maxSize];
        int[] arrC = new int[maxSize];

        System.out.println("enter array A elements: ");
        for(int i = 0; i < maxSize; i++) {
            arrA[i] = sc.nextInt();
        }

        System.out.println("enter array B elements: ");
        for(int j = 0; j < maxSize; j++) {
            arrB[j] = sc.nextInt();
        }

        for (int m = 0; m < maxSize; m++) {
            arrC[m] = arrA[m] + arrB[m];
        }

        System.out.println("Resultant Array: ");
        for(int x = 0; x < arrC.length; x++) {
            System.out.println(arrC[x] + " ");
        }
    }
}